src/video-grid/useCallFeed.js

```diff
@@ -22,9 +22,6 @@ function getCallFeedState(callFeed) {
     member: callFeed ? callFeed.getMember() : null,
     isLocal: callFeed ? callFeed.isLocal() : false,
     speaking: callFeed ? callFeed.isSpeaking() : false,
-    noVideo: callFeed
-      ? !callFeed.stream || callFeed.stream.getVideoTracks().length === 0
-      : true,
```

Comment:
We already do this here: https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/webrtc/callFeed.ts#L170